public/app/features/alerting/unified/components/receivers/grafanaAppReceivers/onCall/onCall.ts

```diff
@@ -0,0 +1,17 @@
+import { Receiver } from '../../../../../../../plugins/datasource/alertmanager/types';
+
+export const isInOnCallIntegrations = (url: string, integrationsUrls: string[]) => {
+  return integrationsUrls.findIndex((integrationUrl) => integrationUrl === url) !== -1;
+};
+
+export const isOnCallReceiver = (receiver: Receiver, integrationsUrls: string[]) => {
+  if (!receiver.grafana_managed_receiver_configs) {
+    return false;
+  }
+  const onlyOneIntegration = receiver.grafana_managed_receiver_configs.length === 1;
```

Comment:
I think we should add a comment here why we care about there only being exactly one integration in the contact point.